h/views/api/flags.py

```diff
@@ -18,19 +19,32 @@
 def create(context, request):
     request.find_service(name="flag").create(request.user, context.annotation)
 
-    _email_group_admin(request, context.annotation)
+    _email_group_moderators(request, context.annotation)
 
     return HTTPNoContent()
 
 
-def _email_group_admin(request, annotation):
+def _email_group_moderators(request, annotation):
     incontext_link = links.incontext_link(request, annotation)
     if incontext_link is None:
         incontext_link = annotation.target_uri
 
-    group = annotation.group
-    if group.creator and group.creator.email:
+    group_members_service = request.find_service(name="group_members")
+
+    memberships = group_members_service.get(
+        annotation.group,
+        roles=[
+            GroupMembershipRoles.OWNER,
+            GroupMembershipRoles.ADMIN,
+            GroupMembershipRoles.MODERATOR,
+        ]
+    )
```

Comment:
This isn't ideal: it hard-codes the logic that owners, admins and moderators are the ones who're allowed to moderate (hide) annotations, duplicating [the same logic in the permission map](https://github.com/hypothesis/h/blob/8765297d0f6c1c3df82d8d64087dd2663d77ccb1/h/security/permission_map.py#L58) (which currently says that only the group's creator can moderate so they're out of sync, that needs to be updated).

The alternative would be to fetch _all_ of the group's members and call `identity_permits(identity, context, Permission.Group.MODERATE)` for every one of them (constructing an `identity` for each member). But that might be a non-starter because it'd not only involve fetching all of the group's members from the DB, but constructing an `Identity` for each member, if it's done via `Identity.from_models(user)`, will also [fetch all of each _user_'s memberships (of all groups the user is a member of) from the DB](https://github.com/hypothesis/h/blob/8765297d0f6c1c3df82d8d64087dd2663d77ccb1/h/security/identity.py#L65).

It may be better to live with this duplication.